README.md

```diff
@@ -36,38 +37,33 @@ $ yarn add unsplash-js
 
 ### Fetch
 
-This library depends on [fetch](https://fetch.spec.whatwg.org/) to make requests to the Unsplash API. For environments that don't support fetch, you'll need to provide polyfills of your choosing. Here are the ones we recommend:
-
-- node implementation: [node-fetch](https://github.com/bitinn/node-fetch)
-- browser polyfill: [whatwg-fetch](https://github.com/github/fetch)
+This library depends on [fetch](https://fetch.spec.whatwg.org/) to make requests to the Unsplash API. For environments that don't support fetch, you'll need to provide polyfills of your choosing. We recommend using [isomorphic-unfetch](https://github.com/developit/unfetch/tree/master/packages/isomorphic-unfetch).
 
 #### Adding polyfills
 
 `createApi` receives an optional `fetch` parameter. When it is not provided, we rely on the globally scoped `fetch`.
 
-This means that you can set the polyfills in the global scope:
+This means that you can set the polyfill in the global scope:
 
 ```ts
-// server
-import fetch from 'node-fetch';
-global.fetch = fetch;
+import 'isomorphic-unfetch';
 
-// browser
-import 'whatwg-fetch';
+// in a browser env, this will set window.fetch = fetch;
+// in a node env, this will set global.fetch = fetch;
 ```
 
-or explicitly provide them as an argument:
+or as a [ponyfill](https://github.com/sindresorhus/ponyfill) by explicitly providing it as an argument:
 
 ```ts
-import nodeFetch from 'node-fetch';
+import fetch from 'isomorphic-unfetch';
 
 const unsplash = createApi({
   accessKey: 'MY_ACCESS_KEY',
-  fetch: nodeFetch,
+  fetch,
 });
 ```
 
-Note: we recommend using a version of `node-fetch` higher than `2.4.0` to benefit from Brotli compression.
+Note: If you choose to use [node-fetch](https://github.com/node-fetch/node-fetch), we recommend using a version higher than `2.4.0` to benefit from Brotli compression. In addition, you may need to import an `AbortController` to patch TypeScript-related issues.
```

Comment:
> In addition, you may need to import an `AbortController` to patch TypeScript-related issues.

Is this referring to the other comment you left in the PR description (below)?

> Additionally, any version > 3 for a TS user would currently see an error like when making a request:
> 
> ```
> message: 'Expected signal to be an instanceof AbortSignal',
> ```

If so, I'm not sure I understand what the problem is. It looks like the error message you mentioned is a runtime error, thrown by `node-fetch`, not a TS (compile time) type error?

Perhaps you could provide a reduced test case of the problem to help me understand?

I tried copying the example from our docs:

```ts
import 'isomorphic-unfetch';

import { createApi } from 'unsplash-js';

const unsplash = createApi({
    accessKey: 'MY_ACCESS_KEY',
});

const controller = new AbortController();
const signal = controller.signal;

unsplash.photos.get({ photoId: '123' }, { signal }).catch((err) => {
    if (err.name === 'AbortError') {
        console.log('Fetch aborted');
    }
});

controller.abort();
```

Using TS 4 I have no type errors. When I try to run this in Node I do get a _runtime_ error, but it is different from the one you mentioned:

```
const controller = new AbortController();
                   ^

ReferenceError: AbortController is not defined
```

For that, I think we should add a note to the docs that an `AbortController` polyfill/ponyfill is required in Node (separate to the changes in this PR).

Reply:
Sorry about that - I shouldn't work at night. You're correct, it's a run-time error, not a TS error.

The confusing part to this is the current behaviors between node-fetch 2.6 and 3, and I was trying to account for the edge cases. (I tested isomorphic-unfetch, cross-fetch, node-fetch with and without abort controllers)

Thanks again for your review, I'll go ahead and do a cleanup pass on this shortly.